internal/hammer/loadtest/hammer.go

```diff
@@ -31,27 +31,35 @@ type HammerOpts struct {
 	NumReadersRandom int
 	NumReadersFull   int
 	NumWriters       int
+	NumMMDVerifiers  int
+	MMDDuration      time.Duration
 }
 
 func NewHammer(tracker *client.LogStateTracker, f client.EntryBundleFetcherFunc, w LeafWriter, gen func() []byte, seqLeafChan chan<- LeafTime, errChan chan<- error, opts HammerOpts) *Hammer {
 	readThrottle := NewThrottle(opts.MaxReadOpsPerSecond)
 	writeThrottle := NewThrottle(opts.MaxWriteOpsPerSecond)
 
+	leafMMDChan := make(chan LeafMMD, opts.NumWriters*opts.NumMMDVerifiers)
```

Comment:
This is a unique way of picking the capacity of the channel. The good news is that it sets the capacity to 0 if there are no verifiers, but I don't like the implication that we're trying to create a channel big enough so that every writer can send a message to every verifier.

Reply:
I've reverted it to use the number of writers. If the number of MMD verifiers is 0, the `leafMMDChan` is `nil` instead of buffer size `0`. The `cap(leafMMDChan)` returns 0 when `leafMMDChan = nil`.

Reply:
I've tracked the number of entries in the channel. It does not reach the number of writers * 12 when there are enough MMD verifiers to process the channel.